### PR TITLE
bmqstoragetool UTs/mqbs_filestore/mqbs_filestoreprotocol: fix uninitialized memory

### PR DESCRIPTION
**Pull request by alexander-e1off**

### CODE REVIEW

File: src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp [hunk 1]
```diff
@@ -57,7 +57,14 @@ using namespace TestUtils;
 //                                    TESTS
 // ----------------------------------------------------------------------------
 
-static void test1_breathingTest()
+// This test has MemorySanitizer false positive, disable uninitialized checks.
```
Comment (678098) [inline src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp:60]: Are you sure that this is a false positive?

**Review by 678098 — COMMENTED** [review]
Just one question

File: src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp (hunk 1 above)
Comment (alexander-e1off) [reply src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp:60]: `MemorySanitizer` complains:
```
==42275==WARNING: MemorySanitizer: use-of-uninitialized-value
#0 0x55a742e52178 in test1_breathingTest() src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp:102:13
```
It is  [here](https://github.com/bloomberg/blazingmq/blob/main/src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp#L102), and `records` list is initialized [here](https://github.com/bloomberg/blazingmq/blob/main/src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp#L78). So, it seems that  `recordIter->first` cannot be uninitialized.

**Review by 678098 — APPROVED** [review]
LGTM

**Pull request merged**